core/src/main/java/com/sequenceiq/cloudbreak/job/StackStatusCheckerJob.java

```diff
@@ -156,7 +156,9 @@ protected void executeInternal(JobExecutionContext context) throws JobExecutionE
                 Status.START_FAILED,
                 Status.STOPPED,
                 Status.STOP_FAILED,
-                Status.AMBIGUOUS
+                Status.AMBIGUOUS,
+                Status.BACKUP_FINISHED,
+                Status.RESTORE_FINISHED
```

Comment:
I don't know which pr will contain the final setting for this, but the 2 FAILED statuses should be here also.
cc @hreeve-cloudera

Reply:
will take care of it.